styles.css

```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto+Serif:opsz,wght@8..144,100&display=swap');
+
+main {
+  padding: 2px;
+}
```

Comment:
- [ ] Your application does not fill the entire mobile page. It has blank spaces around it on browsers. Kindly resize your application to make it looks more like the Figma template and more professional so it can catch more attention from a possible recruiter.
 - [ ] Also try applying the background image and decorations of the Figma template to your app. This can make your app looks more beautiful.
 -Your design:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/61919617/186484913-4083ae55-91bd-4cae-9b35-a96ee8e7a625.png">
-Figma template:
<img width="365" alt="image" src="https://user-images.githubusercontent.com/61919617/186485018-07e1e3dd-b7af-473e-b092-66f674236bb5.png">